src/gui/GuiTools.cpp

```diff
@@ -66,6 +67,19 @@ namespace GuiTools
         }
     }
 
+    size_t expireEntries(QWidget* parent, const QList<Entry*>& entries)

```

Comment:
I also just realized this was in GuiTools. There is no need for a parent in this call and this function can be moved to either a static function of Entry, a static function of Database class, or within core/Tools.

Reply:
My memory of what I intended to do here is coming back to me -- I wanted to create a confirmation dialog similar to what the deletion operation gives you and kinda stopped halfway because I wanted to make sure I was going down the right path first. 

If you're okay with it, I'll remove the parent, move this function to `Database::expireEntries(const QList<Entry*>& entries)`, then create a confirmation interaction similar to the one the delete operation has in `DatabaseWidget::expireEntries()`. `DatabaseWidget::expireEntries()` would remain and call a `GuiTools::confirmExpireEntries()` method that would call the new `Database::expireEntries()` static function, like the delete operation does. Does that sound okay?

Reply:
Yes thatz perfect and allows us to expose this to the cli easily